test/battle/item_effect/escape.c

```diff
@@ -0,0 +1,49 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gItems[ITEM_POKE_TOY].battleUsage == EFFECT_ITEM_ESCAPE);
+}
+
+WILD_BATTLE_TEST("Poke Toy lets the player escape from a wild battle")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { USE_ITEM(player, ITEM_POKE_TOY); }
+    } SCENE {
+        MESSAGE("{PLAY_SE SE_FLEE}Got away safely!\p");
+    }
+}
+
+WILD_BATTLE_TEST("Poke Toy lets the player escape from a wild battle even if a move forbid them to")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(opponent, MOVE_MEAN_LOOK); }
```

Comment:
Sorry, missed that one. Please add an assume for the mean look effect inside this test.

Reply:
Done.